source/operations/troubleshooting.rst

```diff
@@ -164,11 +164,11 @@ For more details about encrypting or decrypting such files, see :ref:`Encrypting
 Logs
 ----
 
-Use the subcommands for ``mc support logs`` to :mc:`~mc support logs enable` or :mc:`~mc support logs disable` the submission of MinIO logs to SUBNET.
-You can also use :mc:`mc support logs status` to check if a log submission is in progress.
+Use the subcommands for ``mc support callhome`` to :mc:`~mc support callhome enable` or :mc:`~mc support callhome disable` the submission of MinIO logs to SUBNET.
+You can also use :mc:`mc support callhome status` to check if a log submission is in progress.
```

Comment:
I'd just drop the lead and say something like

```suggestion
Use :mc:`mc support callhome enable` to start automatically uploading deployment logs to SUBNET for analysis.
Use :mc:`mc support callhome status` to check the status of log upload.
You can disable upload at any time using :mc:`mc support callhome disable`.
```

Reply:
Good suggestion. Done.